enterprise/admin/views.py

```diff
@@ -199,8 +200,19 @@ def _enroll_users(cls, emails, course_details, mode, request):
             ).format(enrolled_count=enrolled_count, course_id=course_id))
         if non_existing:
             messages.warning(request, _(
-                "The following users do not have accounts yet and were not enrolled: {}"
+                "The following users do not have accounts yet and were not yet enrolled: {}"
```

Comment:
OK, makes sense. 

Is this better?

The following users do not have an account on `<platform name>`. They have not been enrolled in the course.